CMakeLists.txt

```diff
@@ -323,6 +323,22 @@ set(SCRIPT_DIR "${PROJECT_SOURCE_DIR}/scripts/")
 # install the module files
 install(DIRECTORY "${PROJECT_SOURCE_DIR}/modules/" DESTINATION ${YANG_MODULE_DIR})
 
+# find sysrepoctl to be used for installation and tests
+if (NOT SYSREPOCTL_EXECUTABLE)
+    find_program(SYSREPOCTL_EXECUTABLE sysrepoctl)
+endif()
+if (NOT SYSREPOCTL_EXECUTABLE)
+    message(FATAL_ERROR "Unable to find sysrepoctl, set SYSREPOCTL_EXECUTABLE manually.")
+endif()
+
+# find sysrepocfg to be used for installation and tests
+if (NOT SYSREPOCFG_EXECUTABLE)
+    find_program(SYSREPOCFG_EXECUTABLE sysrepocfg)
+endif()
+if (NOT SYSREPOCFG_EXECUTABLE)
+    message(FATAL_ERROR "Unable to find sysrepocfg, set SYSREPOCFG_EXECUTABLE manually.")
+endif()
+
```

Comment:
Does not seem to me to be the right place for this code. Either put it after `# sysrepo` in the dependency search some lines above or before `if(INSTALL_MODULES)` below, if interpreting it as being needed only for the scripts.

Reply:
It's needed for both tests and scripts, so I put after the find_package stuff